packages/noya-state/src/selectors/__tests__/__snapshots__/gradientSelectors.test.ts.snap

```diff
@@ -0,0 +1,65 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`get Gradient Point angular gradient 1`] = `
+Array [
+  Object {
+    "color": Object {
+      "_class": "color",
+      "alpha": 1,
+      "blue": 1,
+      "green": 1,
+      "red": 1,
+    },
+    "point": Object {
+      "x": 100,
+      "y": 50,
+    },
+  },
+  Object {
+    "color": Object {
+      "_class": "color",
+      "alpha": 1,
+      "blue": 0,
+      "green": 0,
+      "red": 0,
+    },
+    "point": Object {
+      "x": 100,
+      "y": 49.999999999999986,
+    },
+  },
+]
+`;
+
+exports[`get Gradient Point linear gradient / radial gradient 1`] = `
+Array [
+  Object {
+    "color": Object {
+      "_class": "color",
+      "alpha": 1,
+      "blue": 1,
+      "green": 1,
+      "red": 1,
+    },
+    "point": Object {
+      "x": 50,
+      "y": 0,
+    },
+  },
+  Object {
+    "color": Object {
+      "_class": "color",
+      "alpha": 1,
+      "blue": 0,
+      "green": 0,
+      "red": 0,
+    },
+    "point": Object {
+      "x": 50,
+      "y": 100,
+    },
+  },
+]
+`;
+
+exports[`get Gradient Point sorted points 1`] = `undefined`;
```

Comment:
Should this one have values? We should have an unsorted gradient as input, and double check that it's sorted here.